cmake/Package.cmake

```diff
@@ -83,8 +83,10 @@ if(ANDROID AND ANDROIDDEPLOYQT_EXECUTABLE)
     set(ANDROID_TEMPLATE_FOLDER "${CMAKE_BINARY_DIR}/android-template")
     file(COPY ${CMAKE_SOURCE_DIR}/platform/android/ DESTINATION ${ANDROID_TEMPLATE_FOLDER}/)
     set(SRC_FOLDER "${ANDROID_TEMPLATE_FOLDER}/src/ch/opengis/${APP_PACKAGE_NAME}")
-    file(REMOVE_RECURSE ${SRC_FOLDER}) # remove any pre-existing content
-    file(RENAME "${ANDROID_TEMPLATE_FOLDER}/src/ch/opengis/qfield" ${SRC_FOLDER})
+    if (NOT APP_PACKAGE_NAME EQUAL "qield")
```

Comment:
Typo?

Reply:
@m-kuhn , you must have missed the memo about the app renamed to Qield? ;-P

Reply:
Indeed missed that 😅

One more thing: `STREQUAL` would be correct here, `EQUAL` is optimized for numbers.